graylog2-server/src/main/java/org/graylog2/outputs/BlockingBatchedESOutput.java

```diff
@@ -181,6 +187,11 @@ public void stop() {
         super.stop();
     }
 
+    @Override
+    public void initialize() throws Exception {
+        flushService.scheduleAtFixedRate(this::forceFlushIfTimedout, outputFlushInterval, outputFlushInterval, TimeUnit.SECONDS);
```

Comment:
This doesn't implement any error handling and logging. We should implement the same handling and logging we had in the periodical.

Reply:
yeah, makes sense :+1: